src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown

```diff
@@ -0,0 +1,193 @@
+    
+    <a href='./images/general-walkthrough/search-form.png'><img style="max-width:300px" src="./images/general-walkthrough/search-form.png"/></a>
+    
+    Sample search request:
+
+    ```
+    <Criteria>
+      <Criterion>
+        <Position Latitude="47.606209" Longitude="-122.332071"></Position>
+        <RefPoint></RefPoint>
+        <HotelRef HotelName="Downtown"></HotelRef>
+        <Radius Distance="5" DistanceMax="30" UnitOfMeasureCode="1"></Radius>
+        <StayDateRange Start="2021-09-28" End="2021-10-01"></StayDateRange>
+      </Criterion>
+    </Criteria>
+    ```
+    
+    [Sample search request](./sample-requests/general-walkthrough/search-rq.xml)
+
+    [Sample search response](./sample-requests/general-walkthrough/search-rs.xml)
+
+    Out of 100 returned hotels in response from the Hotel Supplier first 10 hotels are Most Preferred hotels within the 30km radius. The next 10 hotels are Preferred hotels from 30km radius. The remaining 80 hotels are hotels with no preference within the 5km radius. Note: The preference level is defined by the HotelPreference element in the TPA_Extensions, which is outlined in Search.
+
+    More details about the `Search` requests [here](./faq#search)
+
+2. Once the [Search](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Search.html) response is received, an [Availability](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Availability.html) request is invoked. The number of hotel codes specified in the Availability request is determined by the travel configuration setting.
+
+    ````
+    <AvailRequestSegments>
+      <AvailRequestSegment>
+        <HotelSearchCriteria>
+          <Criterion>
+            <HotelRef ChainCode="PF" HotelCode="HP070213"></HotelRef>
+          </Criterion>
+          <Criterion>
+            <HotelRef ChainCode="ZZ" HotelCode="HH910312"></HotelRef>
+          </Criterion>
+          <Criterion>
+            <HotelRef ChainCode="TL" HotelCode="HT030025"></HotelRef>
+          </Criterion>
+          <Criterion>
+            <HotelRef ChainCode="CU" HotelCode="HH871486"></HotelRef>
+          </Criterion>
+          <Criterion>
+            <HotelRef ChainCode="LQ" HotelCode="HL031350"></HotelRef>
+          </Criterion>
+        </HotelSearchCriteria>
+      </AvailRequestSegment>
+    </AvailRequestSegments>
+    ````
+
+    [Sample availability request](./sample-requests/general-walkthrough/avail-rq.xml)
+
+    [Sample availability response](./sample-requests/general-walkthrough/avail-rs.xml)
+
+    More details about the `Availability` requests [here](./faq#availability)
+
+
+3. When the [Availability](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Availability.html) response is received, the UI displays the available hotels. The business traveler can then select View Rooms for a hotel with a lead rate displayed, or request to Get Rates in case they are not present. 
```

Comment:
```suggestion
3. When the [Availability](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Availability.html) response is received, the UI displays the available hotels. The business traveler can then select **View Rooms** for a hotel with a lead rate displayed, or request to **Get Rates** in case they are not present.
```